templates/admin_assets/stylesheets/admin.sass

```diff
@@ -0,0 +1,6 @@
+@import normalize-rails
```

Comment:
ja si to spletl s modernizrem. neni potreba, muzes vyhodit z gemfilu aj odtud. modernizr bych dal taky asi jako optional..